r-ver/3.1.3/Dockerfile

```diff
@@ -0,0 +1,148 @@
+FROM debian:jessie
+
+LABEL org.label-schema.license="GPL-2.0" \
+      org.label-schema.vcs-url="https://github.com/rocker-org/rocker-versioned" \
+      org.label-schema.vendor="Rocker Project" \
+      maintainer="Carl Boettiger <cboettig@ropensci.org>" \
+      contributor="Colin Fay <contact@colinfay.me>"
+
+ARG R_VERSION
+ARG BUILD_DATE
+ENV BUILD_DATE ${BUILD_DATE:-2015-03-09}
+ENV R_VERSION=${R_VERSION:-3.1.3} \
+    LC_ALL=en_US.UTF-8 \
+    LANG=en_US.UTF-8 \
+    TERM=xterm
+
+## dependencies
+# Update the sources.list to archive
+RUN echo 'deb http://archive.debian.org/debian/ wheezy main non-free contrib' \
+  >> /etc/apt/sources.list && \
+  echo 'deb-src http://archive.debian.org/debian/ wheezy main non-free contrib' \
+  >> /etc/apt/sources.list && \
+  echo 'deb http://archive.debian.org/debian-security/ wheezy/updates main non-free contrib' \
+  >> /etc/apt/sources.list && \
+  echo 'deb-src http://archive.debian.org/debian-security/ wheezy/updates main non-free contrib' \
```

Comment:
um, is it possible to get this to build either all on wheezy or all on jessie rather than mixing the two?  [Debian advises against mixing two different stable releases](https://wiki.debian.org/DontBreakDebian#Don.27t_make_a_FrankenDebian), and it can be a pain since they ship with different versions of stuff like gcc and libc which can make their binaries incompatible...  maybe a good reason to do it this way but worth discussing a bit first.

Reply:
That's a mistake from my part. 
I wrote a first version of the Dockerfile with wheezy to experiment, then copied/pasted a clean version to rewrite it properly... and forgot to change the `FROM` this second time. 

I will change it back.